pipelines/batch/setup_prod_job.py

```diff
@@ -78,6 +79,12 @@ def main(
             f"supported diseases are: {', '.join(supported_diseases)}"
         )
 
+    pyrenew_hew_output_container = (
+        "pyrenew-test-output" if test else "pyrenew-hew-prod-output"
+    )
+    n_warmup = 10 if test else 1000
```

Comment:
I think we may as well draw enough samples to make results are interpretable, even in test runs. Sampling time unlikely to be the main bottleneck in running this and it makes the test stronger.